WIP
--> Will run: [ 'runtime', 'prepare', 'install', 'deploy', 'restart', 'show' ]
--> {hostname} - runtime
--> {hostname} - prepare
Found 'terraform' with version '0.6.6'
Found 'terraformInventory' with version '0.5.0'
Found 'pip' with version '7.1.2'
Found 'ansible' with version '1.9.2'
--> {hostname} - install

PLAY [Install freytest-web] *************************************************** 

GATHERING FACTS *************************************************************** 
<107.170.43.164> ESTABLISH CONNECTION FOR USER: root
<107.170.43.164> REMOTE_MODULE setup
<107.170.43.164> EXEC ssh -C -tt -vvv -o User=root -o ControlMaster=auto -o ControlPersist=60s -o ControlPath="{home}/.ansible/cp/ansible-ssh-%h-%p-%r" -o IdentityFile="{root}/test/scenario/digitalocean/frey-digitalocean.pem" -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o User=root -o ConnectTimeout=10 107.170.43.164 /bin/sh -c 'mkdir -p $HOME/.ansible/tmp/ansible-tmp-1447172462.54-162357274514809 && echo $HOME/.ansible/tmp/ansible-tmp-1447172462.54-162357274514809'
fatal: [107.170.43.164] => SSH Error: command-line line 0: garbage at end of line; "-o".
It is sometimes useful to re-run the command using -vvvv, which prints SSH debug output to help diagnose the issue.

TASK: [Common | Add US APT Mirrors] ******************************************* 
FATAL: no hosts matched or all hosts have already failed -- aborting


PLAY RECAP ******************************************************************** 
           to retry, use: --limit @{home}/install.retry

107.170.43.164             : ok=0    changed=0    unreachable=1    failed=0   


--> Exiting with error: Script '{home}/.frey/tools/pip/bin/ansible-playbook -vvvv --private-key={root}/test/scenario/digitalocean/frey-digitalocean.pem --inventory-file={home}/.frey/tools/terraform-inventory --user=root --sudo {root}/test/scenario/digitalocean/.frey/residu/install.yml' exited with code: '3'
--> Details:


PLAY [Install freytest-web] *************************************************** 

GATHERING FACTS *************************************************************** 
<107.170.43.164> ESTABLISH CONNECTION FOR USER: root
<107.170.43.164> REMOTE_MODULE setup
<107.170.43.164> EXEC ssh -C -tt -vvv -o User=root -o ControlMaster=auto -o ControlPersist=60s -o ControlPath="{home}/.ansible/cp/ansible-ssh-%h-%p-%r" -o IdentityFile="{root}/test/scenario/digitalocean/frey-digitalocean.pem" -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o User=root -o ConnectTimeout=10 107.170.43.164 /bin/sh -c 'mkdir -p $HOME/.ansible/tmp/ansible-tmp-1447172462.54-162357274514809 && echo $HOME/.ansible/tmp/ansible-tmp-1447172462.54-162357274514809'
fatal: [107.170.43.164] => SSH Error: command-line line 0: garbage at end of line; "-o".
It is sometimes useful to re-run the command using -vvvv, which prints SSH debug output to help diagnose the issue.

TASK: [Common | Add US APT Mirrors] ******************************************* 
FATAL: no hosts matched or all hosts have already failed -- aborting


PLAY RECAP ******************************************************************** 
           to retry, use: --limit @{home}/install.retry

107.170.43.164             : ok=0    changed=0    unreachable=1    failed=0   



